controllers/eventhandlers/grpcroute.go

```diff
@@ -0,0 +1,89 @@
+package eventhandlers
+
+import (
+	"context"
+	"github.com/golang/glog"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/equality"
+	"k8s.io/apimachinery/pkg/types"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/handler"
+	"sigs.k8s.io/controller-runtime/pkg/reconcile"
+	gateway_api_v1alpha2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
+
+	"k8s.io/client-go/util/workqueue"
+	"sigs.k8s.io/controller-runtime/pkg/event"
+)
+
+type enqueueRequestsForGRPCRouteEvent struct {
+	client client.Client
+}
+
+func NewEnqueueRequestGRPCRouteEvent(client client.Client) handler.EventHandler {
+	return &enqueueRequestsForGRPCRouteEvent{
+		client: client,
+	}
+}
+
+func (h *enqueueRequestsForGRPCRouteEvent) Create(e event.CreateEvent, queue workqueue.RateLimitingInterface) {
+	glog.V(6).Info("GRPCRoute create")
```

Comment:
Be more specific with logging, add something about event you receive. Same for other logs without specifics.

Reply:
Will do!